.github/workflows/climpred_installs.yml

```diff
@@ -0,0 +1,22 @@
+name: climpred installs
```

Comment:
Worth adding this. Found out if you install `climpred` from scratch it doesn't work without `ipython` and `toolz` installs.